Could you please help fix the [PyLint error](https://github.com/tensorflow/tensorflow/actions/runs/3949051506/jobs/6759788729)?
```
tensorflow/dtensor/python/tests/test_util.py:278:0: C0301: Line too long (81/80) (line-too-long)
```

--- a/tensorflow/dtensor/python/tests/test_util.py
+++ b/tensorflow/dtensor/python/tests/test_util.py
@@ -275,7 +275,7 @@ class DTensorBaseTest(tf_test.TestCase, parameterized.TestCase):
       self.assertEqual(
           api.fetch_layout(result_dtensor),
           expected_layout,
-          msg='========\nexpected layout is\n  {}\n\nwhile got layout is\n  {}\n'
+          msg='=======\nexpected layout is\n  {}\n\nwhile got layout is\n  {}\n'
           .format(expected_str, got_str))
 
     layout = api.fetch_layout(result_dtensor)

